### ISSUE TITLE
XSplit Broadcaster output files causing timeline to bug at 1 second - URGENT appeal, YT video due to go up

### ISSUE THREAD

**Issue by itschris2003** [OpenShot/openshot-qt#973]
**System Details:**
* Operating System / Distro: Windows 7
* OpenShot Version: 2.4.0

Hello all, my openshot seems to have bugged out. When I import mp4 videos i previously recorded in xSplit Broadcaster and I zoom into the timeline for 1 seconds (to get all video/audio positions exact as possible), the timeline seems to disappear and when I preview the video, it's just black with no audio... Help?

**Comment by itschris2003** [comment]
#

**Comment by DylanC** [comment]
@itschris2003 - Did you try this in v2.4.1?

**Comment by itschris2003** [comment]
@DylanC yes, just did. all seems to be fine. Issue therefore is resolved (i think xD)

**Comment by DylanC** [comment]
@itschris2003 - Thanks.
